js/cellGenerator.mjs

```diff
@@ -74,27 +74,19 @@ function generateLinkUrl(cell, format){
             return null;
         }
         else{
-            linkHref = "./map.html?formId="+cell.formId;
+            linkHref = "./map.html?formId=" + cell.formId;
             if((format & CELL_FORMAT_NAMELINK_OPEN_IN_IFRAME) && window.settings.iframeCheck == "on"){
```

Comment:
change this iframeCheck to != "off" instead of forcing topbar to false for all maps.

Reply:
I think I did it correctly, should be fixed now.